src/js/helpers/get-rendition.js

```diff
@@ -28,7 +28,7 @@ function getRendition(renditions, options) {
 		return false;
 	});
 
-	return appropriateRendition || orderedRenditions.shift();
+	return appropriateRendition || orderedRenditions.pop();
```

Comment:
You added one already, great work! (https://github.com/Financial-Times/o-video/pull/29/files#diff-fb856e7d307df4c5b3a7f1aa31c40683R29)